public/scotland/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "BBC Scotland",
+  "short_name": "BBC Scotland",
+  "Scope": "/scotland/",
+  "background_color": "#23104C",
+  "display": "standalone",
+  "theme_color": "#23104C",
+  "icons": [
+    {
+      "src": "https://news.files.bbci.co.uk/include/articles/public/scotland/images/icons/icon-72x72.png",
```

Comment:
Can you please also add the icons in this directory: `public/scotland/images/icons/`? When the PR is merged, the images will be uploaded to the appropriate S3 bucket and will be accessible at these endpoints.

Reply:
Can these be added as a subsequent PR so as not to block this?

Reply:
I've written https://github.com/bbc/simorgh/issues/3845 to cover adding these so as not to block this PR progressing.